src/main/kotlin/com/jfrog/conan/clionplugin/conan/Conan.kt

```diff
@@ -20,7 +40,7 @@ class Conan (val project: Project) {
     }
 
     fun install(name: String, version: String): String {
-        val args = "install $name/$version -r conancenter".split(" ").toList()
+        val args = "install --requires=$name/$version -r=conancenter --build=missing".split(" ").toList()
```

Comment:
added also the correct syntax with --requires and --build=missing but we should think if we want this as something you can configure